CMakeLists.txt

```diff
@@ -1,5 +1,4 @@
 cmake_minimum_required(VERSION 3.4)
-cmake_policy(SET CMP0037 NEW)
```

Comment:
3.4 uses OLD if not set: https://cmake.org/cmake/help/v3.4/policy/CMP0037.html

Reply:
Won't merge this one.

Reply:
If you do a cmake_minimum_required(VERSION X) then all policies introduced in all prior versions are automatically set to NEW. CMP0037 was introduced in 3.0, so it already is NEW here.

Reply:
:cherries: -picked